pkg/daemon/command/command.go

```diff
@@ -122,12 +124,23 @@ func WithThreadNum(num int) Opt {
 	}
 }
 
-func WithConfig(config string) Opt {
+func WithConfig(id string) Opt {
```

Comment:
Better not to change the method, so that we can keep compatibility for old nydusd versions.